hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/healthcondition/HealthCondition.scala

```diff
@@ -0,0 +1,105 @@
+package org.broadinstitute.monster.dap.healthcondition
+
+import enumeratum.values.{LongEnum, LongEnumEntry}
+
+sealed abstract class HealthCondition(
+  override val value: Long,
+  val label: String,
+  val conditionType: HealthConditionType,
+  val abbreviation: String,
```

Comment:
I am making an assumption here that the "core" abbreviation for a disease is consistent across cg/dx cases, even if the prefix/suffix vary. It's true for all the cases we'd already coded / for other cases I spot-checked.